packages/hooks/useTimeout.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const useTimeout: Function;
```

Comment:
this is assuming its a `const`when its a function. I also need to refactor the other ones to be correct but below is the correct way:
```typescript
declare function useTimeout(ms: number): boolean;
```